docs/development_suite/webhooks-and-events/events.mdx

```diff
@@ -100,70 +509,356 @@ hence there is no guarantee that the service will be found in the project config
 | `repositoryURL` | `string` | The URL of the repository that hosts the service code                                        |
 | `serviceName`   | `string` | The name of the service inside the project configuration                                     |
 | `templateId`    | `string` | The identifier of the template in the Console marketplace from which the service was created |
+| `service`       | `object` | The service metadata (ideally, the whole service resource)                                   |
 
 <Tabs>
 <TabItem value="example" label="Example" default>
 
 ```json
-  {
-    "eventName": "service_created",
-    "eventTimestamp": 123456,
+{
+  "eventName": "service_created",
+  "eventTimestamp": 123456,
+  "payload": {
+    "tenantId": "my-company-id",
+    "projectId": "0000000000000001",
+    "repositoryURL": "http://gihub.com/my-repository",
+    "serviceName": "my-service",
+    "templateId": "0000000000000002"
+  }
+}
+```
+
+</TabItem>
+<TabItem value="json-schema" label="JSON Schema">
+
+```json
+{
+  "additionalProperties": false,
+  "properties": {
+    "eventName": {
+      "const": "service_created",
+      "type": "string"
+    },
+    "eventTimestamp": {
+      "type": "number"
+    },
     "payload": {
-      "tenantId": "my-company-id",
-      "projectId": "0000000000000001",
-      "repositoryURL": "http://gihub.com/my-repository",
-      "serviceName": "my-service",
-      "templateId": "0000000000000002"
+      "additionalProperties": false,
+      "properties": {
+        "tenantId": {
+          "type": "string",
+          "description": "The identifier of the Company the service belongs to"
+        },
+        "projectId": {
+          "type": "string",
+          "description": "The identifier of the Project the service belongs to"
+        },
+        "repositoryURL": {
+          "type": "string",
+          "description": "The URL of the repository that hosts the service code"
+        },
+        "serviceName": {
+          "type": "string",
+          "description": "The name of the service inside the project configuration"
+        },
+        "templateId": {
+          "type": "string",
+          "description": "The identifier of the template in the Console marketplace from which the service was created"
+        },
+        "service": {
+          "type": "object",
+          "description": "The service metadata (ideally, the whole service resource)"
+        }
+      },
+      "required": ["projectId", "serviceName", "repositoryURL", "templateId"],
+      "type": "object"
     }
+  },
+  "required": ["eventName", "eventTimestamp", "payload"],
+  "type": "object"
 }
 ```
 
 </TabItem>
-<TabItem value="json-schema" label="JSON Schema">
+<TabItem value="service-json-schema" label="service JSON Schema">
 
 ```json
 {
-    "additionalProperties": false,
-    "properties": {
-      "eventName": {
-        "const": "service_created",
-        "type": "string"
+  "type": "object",
```

Comment:
From the documentation point of view, could we simplify the json schema? And, maybe, we could make an event example AND the json schema (or also other type of type documentation) for the structure? (also for the project)

Reply:
Yes, we try, but that is really hard to show only one example. There are more cases to cover.

Reply:
we decided to show some example